10.0/s2i/bin/assemble

```diff
@@ -197,20 +276,14 @@ if [ -f "$LOCAL_SOURCE_DIR/pom.xml" ]; then
   popd &> /dev/null
 else
   echo "Copying binaries in source directory into $DEPLOY_DIR for later deployment..."
-  cp $LOCAL_SOURCE_DIR/*.war $DEPLOY_DIR >& /dev/null
-  chgrp -R 0 $DEPLOY_DIR
-  chmod -R g+rw $DEPLOY_DIR
+  copy_artifacts "$LOCAL_SOURCE_DIR" war ear rar jar
 fi
 
 if [ -d $LOCAL_SOURCE_DIR/target ]; then
-  cp $LOCAL_SOURCE_DIR/target/*.war $DEPLOY_DIR >& /dev/null
-  chgrp -R 0 $DEPLOY_DIR
-  chmod -R g+rw $DEPLOY_DIR
+  copy_artifacts "target" war ear rar jar
 fi
 if [ -d $LOCAL_SOURCE_DIR/deployments ]; then
-  cp $LOCAL_SOURCE_DIR/deployments/*.war $DEPLOY_DIR >& /dev/null
-  chgrp -R 0 $DEPLOY_DIR
-  chmod -R g+rw $DEPLOY_DIR
+  copy_artifacts "deployments" war ear rar jar
```

Comment:
@bparees Why?
Permissions are fixed on the LOCAL_SOURCE_DIR (https://github.com/jorgemoralespou/sti-wildfly/blob/a5e07b92aaf1b3a77657705bf2a61839ba411dce/10.0/s2i/bin/assemble#L221-L223) and then artifacts are copied so permissions should be correct for deployments in source Git.

I'll add the fix, but I'm wondering if we're adding a lot of complexity to the assemble just to be sure things work without really validating that things are not working.